chapters/03_queueing_model/01_discrete_event_simulation/main.tex

```diff
@@ -0,0 +1,281 @@
+\subsection{Discrete Event Simulation}
+
+Discrete event simulation (DES) is currently one of the most popular modelling
+techniques.
+It is a method for modelling the behaviour of real-world systems in which the
+system is made up of discrete events, each of which has a certain
+duration~\cite{DESstewart}.
+It can be used to understand complex situations in order to make predictions
+and thus provide improvements~\cite{VinceGeraintBook}.
+There three main approaches to building DES models are the activity scanning
+approach, the event scheduling approach and the process interaction
+approach~\cite{DESapproaches}.
+Under the scope of this study only the event scheduling approach is considered.
+This section describes the discrete event simulation (DES) model used to
+represent the queueing network of Section~\ref{sec:queueing-section}.
+
+In order to use DES on the queueing network described in
+Figure~\ref{fig:diagram_of_queueing_system} some slight modifications are
+required.
+The current queueing network is a two-node queueing system that accepts two
+types of individuals, where type 1 individuals arrive at waiting zone 1 and
+type 2 individuals arrive at waiting zone 2.
+The modification that is required revolves around the mechanisms of waiting
+zone 2.
+Waiting zone 2 is defined as a non-service node where there is only a queueing
+space for individuals to wait there until they are allowed to waiting zone 1.
+From an implementation perspective there is an equivalent system that can be
+used where instead of a node with no service and queueing capacity \(M\), there
+are \(M\) servers each serving with a service rate of \(0\) and no queueing
+capacity.
+
+\input{chapters/03_queueing_model/img/equivalent_queueing_model_diagram.tex}
+
+The arrival times for both waiting zones are exponentially distributed with
+mean \(\lambda_1\) and \(\lambda_2\) corresponding to type 1 and type 2
+individuals respectively.
+Waiting zone 1 has an exponentially distributed service time with rate \(\mu\),
+a total of \(C\) servers and a queueing capacity of \(N - C\) (making the
+overall capacity \(N\)). 
```

Comment:
Here you're using waiting zone but include service rates, would service node not be a better term?

A service node can then be made up of servers and waiting zones.

Reply:
I'm using the term `waiting zone` to describe the entire node but I can see how this can be problematic. I can change these to `server node 1` and `server node 2`. Although no service takes place at `node 2` (deterministic service time of `0`). It's still fine to call it a server node right?

Should I update this diagram as well then:
![image](https://user-images.githubusercontent.com/20097997/185180767-4316b7c9-59dc-4678-b74b-f980416f53e3.png)